### ISSUE TITLE
Admin New User Preferences - Ensure buttons have discernible text

### ISSUE THREAD

**Issue by will0684** [gcdevops/HRWhiteListing#249]
Title: WCAG 4.1.2: Ensures buttons have discernible text (.o_dropdown_button[draggable="false"][role="button"])
Tags: Accessibility, WCAG 4.1.2, button-name

Issue: Ensures buttons have discernible text (button-name - https://dequeuniversity.com/rules/axe/3.5/button-name?application=msftAI)

Target application: Odoo - http://localhost:8069/web#action=74&active_id=1&cids=1&id=&menu_id=95&model=res.users&view_type=form

Element path: div[name="alias_id"] > .o_input_dropdown > .o_dropdown_button[draggable="false"][role="button"]

Snippet: `<a role="button" class="o_dropdown_button" draggable="false"></a>`

<img width="1674" alt="Screen Shot 2020-04-23 at 10 46 51 AM" src="https://user-images.githubusercontent.com/31868510/80113078-d3c47680-854f-11ea-9f0a-333ca334a9f3.png">

How to fix: 
Fix any of the following:
  Element does not have inner text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element's default semantics were not overridden with role="presentation"
  Element's default semantics were not overridden with role="none"
  Element has no title attribute or the title attribute is empty

Environment: Chrome version 80.0.3987.163

====

This accessibility issue was found using Accessibility Insights for Web 2.16.0 (axe-core 3.5.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
